packages/kolibri-tools/lib/cli.js

```diff
@@ -141,7 +141,7 @@ program
     const buildModule = {
       [modes.PROD]: 'production.js',
       [modes.DEV]: 'webpackdevserver.js',
-      [modes.I18N]: 'i18n.js',
+      [modes.I18N]: 'i18n/index.js',
```

Comment:
As I suggested above, I think we can pull this out into a top level `i18n` command alongside `build`, `lint`, and `test`.